final_task/rss_reader/database_functions.py

```diff
@@ -0,0 +1,87 @@
+import logging
+import psycopg2
+
+def readingPasswordFromFile() -> str:
+    return "1234"
+
+def checkExistance() -> bool:
+    con = None
+    flag = True
+    try:
+        con = psycopg2.connect(
+            database="postgres",
+            user="postgres",
+            password=readingPasswordFromFile(),
+            host="127.0.0.1",
+            port="5432"
+        )
+        cur = con.cursor()
+        cur.execute("SELECT * from cache")
+        logging.debug("Table exists")
+    except psycopg2.DatabaseError as error:
+        flag = False
+        logging.error(error)
+        logging.debug("Table doesn't exist")
+    finally:
+        if con is not None:
+            con.close()
+        return flag
```

Comment:
Here and below:

It is better to use your own context manager for connections with the db. (I would advise to implement it as a function)